tools/tkn-results/internal/config/config.go

```diff
@@ -55,19 +56,20 @@ func init() {
 	viper.SetConfigName("results")
 	viper.SetConfigType("yaml")
 	viper.AddConfigPath("$HOME/.config/tkn")
-	pflag.StringP("addr", "a", "", "Result API server address")
-	pflag.StringP("authtoken", "t", "", "authorization bearer token to use for authenticated requests")
-	pflag.Parse()
+	err := setConfig()
+	if err != nil {
+		fmt.Println("error setting up flags and config", err)
```

Comment:
Why don't we `os.Exit(1)` here?

Reply:
I will update with log.Fatal.  Forgot to put it here.

Reply:
Fixed.